open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/SubjectAreaClient.java

```diff
@@ -9,65 +9,191 @@
 
 import java.util.List;
 
+/**
+ * @param <T> Glossary artifact type for supplied and return.
+ * Interface describing common methods to client working with Subject area resources.
+ */
 public interface SubjectAreaClient<T> {
 
     FindRequest EMPTY_FIND_REQUEST = new FindRequest();
 
+    /**
+     * Get Glossary artifact by guid
+     *
+     * @param userId unique identifier for requesting user, under which the request is performed.
+     * @param guid   unique identifier of the Glossary artifact.
+     * @return found Glossary artifacts of the T type.
```

Comment:
Sorry I am being very picky  @immmus artifacts -> artifact. 
Could we should Glossary Artifact with a capital A so it is not seen as a Glossary - in all occurances

I will merge after these changes

Reply:
okay